api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementServiceTest.java

```diff
@@ -103,14 +122,42 @@ void shouldReturnEligibleWhenNotDuplicateAndEligible() {
         EligibilityAndEntitlementDecision decisionResponse = setupEligibilityAndEntitlementDecisionFactory(ELIGIBLE);
 
         //When
-        EligibilityAndEntitlementDecision result = eligibilityAndEntitlementService.evaluateNewClaimant(CLAIMANT);
+        EligibilityAndEntitlementDecision result = eligibilityAndEntitlementService.evaluateNewClaimant(CLAIMANT, NO_ELIGIBILITY_OVERRIDE);
 
         //Then
         assertThat(result).isEqualTo(decisionResponse);
         verifyCommonMocks(NOT_DUPLICATE);
         verify(duplicateClaimChecker).liveClaimExistsForHousehold(IDENTITY_AND_ELIGIBILITY_RESPONSE);
     }
 
+    @Test
+    void shouldReturnEligibilityOverride() {
+        //Given
+        given(paymentCycleEntitlementCalculator.calculateEntitlement(any(), any(), any())).willReturn(VOUCHER_ENTITLEMENT);
+        given(claimRepository.findLiveClaimWithNino(any())).willReturn(Optional.empty());
+        given(duplicateClaimChecker.liveClaimExistsForHousehold(any(CombinedIdentityAndEligibilityResponse.class))).willReturn(NOT_DUPLICATE);
+        EligibilityAndEntitlementDecision decisionResponse = setupEligibilityAndEntitlementDecisionFactory(ELIGIBLE);
+
+        //When
+        EligibilityAndEntitlementDecision result = eligibilityAndEntitlementService.evaluateNewClaimant(CLAIMANT, EligibilityOutcome.CONFIRMED);
+
+        //Then
+        CombinedIdentityAndEligibilityResponse response = CombinedIdentityAndEligibilityResponse.builder()
```

Comment:
I think this would be a good candidate to put in a TestDataFactory as `aConfirmedOverrridenResponse` or `anOverridenResponse` taking in the `EligibilityOutcome`.

Reply:
done